codespell_lib/data/dictionary.txt

```diff
@@ -23939,6 +23939,7 @@ soket->socket
 sokets->sockets
 solarmutx->solarmutex
 solatary->solitary
+solderd->soldered, solder, soldier, soldiered,
```

Comment:
I think we should remove this, because it seems _extremely_ unlikely that someone would make this mistake in a typical project.